FIAT/__init__.py

```diff
@@ -29,7 +28,7 @@
 from FIAT.tensor_product import TensorProductElement
 from FIAT.enriched import EnrichedElement
 from FIAT.discontinuous import DiscontinuousElement
-from FIAT.trace_hdiv import HDivTrace
+from FIAT.hdiv_trace import TraceHDiv
```

Comment:
So @dorugeber's module was called *trace H(div)* and his class *H(div) trace*, while your module is called *H(div) trace* and your class *trace H(div)*. Can we just make up our mind on what the order of *trace* and *H(div)* should be?